test/support/test_helpers.ex

```diff
@@ -36,6 +37,27 @@ defmodule RemoteRetro.TestHelpers do
     Map.put(context, :user, user)
   end
 
+  def persist_other_user_for_retro(context) do
+    context = Map.merge(%{other_user: @other_user}, context)
+    %{other_user: other_user} = context
+    user_params = User.build_user_from_oauth(other_user)
+    other_user =
+      User.changeset(%User{}, user_params)
+      |> Repo.insert!
+
+    Map.put(context, :other_user, other_user)
+  end
+
+  def assign_idea(%{other_user: other_user, retro: retro} = context) do
+    idea = %RemoteRetro.Idea{assignee_id: other_user.id, body: "blurgh", category: "action-item", retro_id: retro.id, user_id: other_user.id} |> RemoteRetro.Repo.insert!
+    Map.put(context, :idea, idea)
+  end
+
+  def set_participation(%{other_user: other_user, retro: retro} = context) do
```

Comment:
given that a participation record represents the link of a user to a retro, i think we might want to encapsulate the creation of a participation in the `persist_users_for_retro` helper. the benefit is that then we don't have to call `set_participation` after we've called the helper that builds the necessary user. Whaddaya think?

Reply:
the only reason not to do this was if we wanted to test users without participations - like potentially if a user was invited but never joined and was assigned an action item?  or something wierd like that?  Not sure.  Might be overly convoluted.  I think I'll just set it in `persist_users` as you suggest and then if its a problem down the road we can split it out.

Reply:
👍 !